pkg/model/components/kubecontrollermanager.go

```diff
@@ -73,7 +73,7 @@ func (b *KubeControllerManagerOptionsBuilder) BuildOptions(o interface{}) error
 	// TLDR; set this too low, and have a few EBS Volumes, and you will spam AWS api
 
 	// if 1.4.8+ and 1.5.2+
-	if kubernetesVersion.GTE(*k8sv148) || kubernetesVersion.GTE(*k8sv152) {
+	if (kubernetesVersion.GTE(*k8sv148) && kubernetesVersion.Minor == 4) || kubernetesVersion.GTE(*k8sv152) {
```

Comment:
Justin is saying that we could hit this logic if we get to Kubernetes `2.4.X`

```
if (kubernetesVersion.GTE(*k8sv148) && (kubernetesVersion.Minor == 4 && kubernetesVersion.Major == 1) || kubernetesVersion.GTE(*k8sv152) 
```
Ought to do it..